core/tests/protractor/learnerFlow.js

```diff
@@ -69,11 +69,12 @@ describe('Learner dashboard functionality', function() {
     explorationEditorMainTab.setContent(forms.toRichText(
       'So what can I tell you?'));
     explorationEditorMainTab.setInteraction('MultipleChoiceInput', [
```

Comment:
@donosco98 I suggest that you don't make any changes to the existing behaviour of setInteraction(). Keep the signature the same and don't pass in editAction.

Instead, make changes only to editInteraction. This will help reduce the churn in this part of the codebase.

So, basically, set the interaction with customization args, add rules, and then only edit the interaction using the new function.

Reply:
done